pkgs/by-name/ai/aic8800-firmware/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "aic8800-firmware";
+  version = "4.0+git20250410.b99ca8b6-2";
+
+  src = fetchFromGitHub {
+    owner = "radxa-pkg";
+    repo = "aic8800";
+    tag = finalAttrs.version;
+    hash = "sha256-ol3FwwzUS3x8YQp8xYZJHqQO8nPmZmJN/TjmqcA9G6g=";
+  };
+
+  doBuild = false;
+
+  buildPhase = ''
+    runHook preBuild
+
+    # do nothing
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/firmware/aic8800_fw/PCIE/
+    mkdir -p $out/lib/firmware/aic8800_fw/SDIO/
+    mkdir -p $out/lib/firmware/aic8800_fw/USB/
+
+    cp -rv src/PCIE/driver_fw/fw/* $out/lib/firmware/aic8800_fw/PCIE/
+    cp -rv src/SDIO/driver_fw/fw/* $out/lib/firmware/aic8800_fw/SDIO/
+    cp -rv src/USB/driver_fw/fw/* $out/lib/firmware/aic8800_fw/USB/
+
+    runHook postInstall
+  '';
+
+  passthru.source = {
+    inherit (finalAttrs) version src;
+  };
```

Comment:
You can just do `aic8800-firmware.version` etc. No need to copy them into passthru.